app/services/proxy_status/tftp.rb

```diff
@@ -0,0 +1,24 @@
+module ProxyStatus
+  class TFTP < Base
+    def initialize(proxy, opts = {})
+      super(proxy, opts, 'TFTP')
+    end
+
+    def tftp_server
```

Comment:
Nitpick, how about just `server`? Think about how it is called, `proxy_status[:tftp].tftp_server` vs `proxy_status[:tftp].server`